save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt

```diff
@@ -74,7 +75,7 @@ class FallbackView : AbstractView<FallbackViewProps, State>(Style.SAVE_LIGHT) {
             }
 
             if (props.withRouterLink == true) {
-                Link {
+                Navigate {
                     to = "/"
```

Comment:
may be create global `homePage = /` ?

Reply:
Not sure what you mean. New `FC`?

Reply:
I guess he means
```kotlin
# Constants.kt
const val homePage = "/"
```

Reply:
oh
not sure if it is needed

Reply:
will merge now, later will fix if needed